pkg-r/vignettes/get-started.Rmd

```diff
@@ -0,0 +1,347 @@
+---
+
+<style>
+.callout {
+  border: 1px solid #dfe1e5;
+  border-left: 4px solid #2680e3;
+  border-radius: 6px;
+  background-color: #fff;
+  margin: 1em 0;
+}
+
+.callout > .title {
+  background-color: #edf4fd;
+  padding: 0.5em 1em;
+  font-weight: bold;
+  border-bottom: 1px solid #dfe1e5;
+}
+
+.callout > .title::before {
+  content: "ℹ️";
+  margin-right: 0.5em;
+}
+
+.callout > .body {
+  padding: 1em;
+}
+
+p.caption {
+  text-align: center;
+  font-size: 0.85em;
+  color: #555;
+  margin-top: 0.75em;
+  margin-bottom: 1em;
+}
+
+img.rounded.shadow {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 0.5rem;
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+}
+</style>
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, echo=FALSE}
+library(shinychat)
+```
+
+This article will cover how to build a chatbot powered by a Large Language Model (LLM) using shinychat and [ellmer](https://ellmer.tidyverse.org/). ellmer will handle connecting to and communicating with the model, while shinychat will handle the user interface for your chatbot. 
+
+You will need to install both shinychat and ellmer. 
+
+```r
+install.packages(c("shinychat", "ellmer"))
+```
+
+## Setup
+
+### Choose a model
+
+First, choose a model to power your chatbot. ellmer and shinychat support a [wide variety](https://ellmer.tidyverse.org/#providers) of LLM providers including Anthropic, OpenAI, Vertex, Snowflake, Groq, Perplexity, and more.
+
+With ellmer, you specify the LLM provider by choosing the corresponding `chat_*()` function, e.g., `chat_anthropic()`,  `chat_openai()`, etc. This makes it easy to swap out the chat provider to a different one at any time. 
+
+Model providers also typically offer a variety of models. To specify a particular model, use the `chat_*()` function's `model` argument. For example:
+
+```r
+chat_openai(model = "o3")
+```
+
+If you don't specify the `model` argument, the `chat_*()` function will use a reasonable default. For more information, see the individual `chat_*()` function's [documentation](https://ellmer.tidyverse.org/reference/index.html#chatbots). 
+
+<div class="callout">
+  <div class="title">Help me choose!</div>
+  <div class="body">
+If you're not sure which provider to choose, ellmer provides a [guide](https://ellmer.tidyverse.org/#providermodel-choice) to help you decide. 
+
+If you’re not ready to sign up for a cloud provider (e.g., OpenAI or Anthropic), you can run models locally (for free!) with [Ollama](https://ellmer.tidyverse.org/reference/chat_ollama.html).
+
+A local model is a good way to get started without incurring any cost or sharing your data with a cloud provider. You will need to [install Ollama](https://ellmer.tidyverse.org/reference/chat_ollama.html) and an open-source model (e.g., Llama 3.1). You can use the [`ollamar` package](https://hauselin.github.io/ollama-r/) to install particular models from R. 
```

Comment:
This feels a bit out of place since the 1st example uses `chat_openai()`. I think you could simply remove or condense+move it closer to where you use `chat_ollama()`